packages/components/FocusZone/macos/RCTFocusZone.m

```diff
@@ -45,6 +45,19 @@ static inline CGFloat GetMinDistanceBetweenRectVerticesAndPoint(NSRect rect, NSP
 	);
 }
 
+static NSView *GetFirstKeyViewWithin(NSView *parentView)
```

Comment:
Would it make sense to have a GetLastKeyViewWithin to get the last menu button for when a user launches a menu and presses ArrowUp?

Reply:
Yes. It would also be helpful to implement keyboard overrides for the 'Home' and 'End' keys (which I found out navigate to the beginning and end of a list .

The behavior of my ContextualMenu vs NSmenu is slightly different. In an NSMenu, you have to ArrowDown to focus on the first item. With FURN ContextualMenu, focus is _already_ on the first item, so no arrow key is required. That was a small enough difference I was willing to keep that change...

Reply:
What adds to the complications is that for submenus the first item always _is_ in focus. Just the root menu it's not...